kitsune/inproduct/admin.py

```diff
@@ -1,13 +1,32 @@
 from django.contrib import admin
+from django.utils.html import format_html
 
 from kitsune.inproduct.models import Redirect
+from kitsune.sumo.urlresolvers import reverse
 
 
 class RedirectAdmin(admin.ModelAdmin):
-    list_display = ("product", "version", "platform", "locale", "topic", "target")
+    list_display = (
+        "product",
+        "version",
+        "platform",
+        "locale",
+        "topic",
+        "target",
+        "target_link",
+    )
     list_display_links = ("topic", "target")
     list_filter = ("product", "version", "platform", "locale")
     search_fields = ("topic", "target")
 
+    @admin.display(description="KB Article")
+    def target_link(self, obj):
+        if "kb" in obj.target:
+            target_url = reverse(
+                "wiki.document", kwargs={"document_slug": obj.target.split("/")[-1]}
+            )
```

Comment:
If a `locale` was specified in the redirect, we should use that locale in the link to the KB article, otherwise it's OK to default to `None` (which means that `reverse` will use `en-US`).
```suggestion
            target_url = reverse(
                "wiki.document",
                kwargs={"document_slug": obj.target.split("/")[-1]},
                locale=obj.locale if obj.locale else None,
            )
```

Reply:
Thanks!